### PR TITLE
1900: edit trial session form and sequences

### PR DESCRIPTION
**Pull request by rachaelparris**
Screen Shot 2019-12-20 at 2 05 31 PM

### CODE REVIEW

File: web-client/src/views/TrialSessions/EditTrialSession.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                <label className="usa-label" htmlFor="notes">
+                  Trial session notes{' '}
+                  <span className="usa-hint">(optional)</span>
+                </label>
+                <textarea
+                  className="usa-textarea"
+                  id="notes"
+                  maxLength="400"
+                  name="notes"
+                  value={form.notes}
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessions/EditTrialSession.jsx:70]: will this need ` form.notes || '' ` ?

**Review by kkoskelin — APPROVED** [review]
Looks awesome.

File: web-client/src/views/TrialSessions/EditTrialSession.jsx (hunk 1 above)
Comment (rachaelparris) [reply web-client/src/views/TrialSessions/EditTrialSession.jsx:70]: I'm not seeing any of those controlled/uncontrolled component errors, so I _think_ it's ok.

File: web-client/src/presenter/actions/TrialSession/updateTrialSessionAction.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * @param {object} providers.applicationContext the application context
+ * @param {Function} providers.get the cerebral get helper function
+ * @param {object} providers.path the next object in the path
+ * @returns {Promise<object>} the next path based on if creation was successful or error
+ */
+export const updateTrialSessionAction = async ({
+  applicationContext,
+  get,
+  path,
+  props,
```
Comment (matthopson) [inline web-client/src/presenter/actions/TrialSession/updateTrialSessionAction.js:18]: missing in jsdocs

File: web-client/src/views/TrialSessions/EditTrialSession.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    return (
+      <>
+        <TrialSessionDetailHeader />
+
+        <section className="usa-section grid-container DocumentDetail">
+          <h1>Edit Trial Session</h1>
+
+          <form
+            noValidate
+            aria-labelledby="start-case-header"
```
Comment (matthopson) [inline web-client/src/views/TrialSessions/EditTrialSession.jsx]: Probably for later, but this doesn't point to anything.

**Review by matthopson — APPROVED** [review]
Couple nits to be addressed at some point.

**Pull request merged**